src/main/python/fine_tuning/run_batch.py

```diff
@@ -207,11 +215,11 @@ def del_method_related_files(method_name):
     parser.add_argument('--anserini_root', default='', help='Anserini path')
     parser.add_argument('--run', action='store_true', help='Generate the runs files and evaluate them. Otherwise we only output the evaluation results (based on the existing eval files)')
     parser.add_argument('--collection', required=True, help='the collection key in yaml')
-    parser.add_argument('--basemodel', default='bm25', choices=['bm25', 'ql'], help='the ranking model')
-    parser.add_argument('--model', default='axiom', choices=['bm25', 'ql', 'axiom', 'rm3', 'bm25+axiom', 'bm25+rm3'], help='the higher level model')
-    parser.add_argument('--n', dest='parallelism', type=int, default=16, help='number of parallel threads for retrieval/eval')
+    parser.add_argument('--model', required=True, help='model')
+    parser.add_argument('--threads', dest='parallelism', type=int, default=16, help='number of parallel threads for retrieval and evaluation')
     parser.add_argument('--output_root', default='fine_tuning_results', help='output directory of all results')
-    parser.add_argument('--use_drr_fold', action='store_true', help='if specified, use the 5-folds from the deep-relevance-ranking paper')
+    parser.add_argument('--fold_settings', default='', help='fold setting')
```

Comment:
Can we make the help text more informational? something like `json file contains the fold settings, format e.g. http://xyz.com/abc.json`

Reply:
Done.